frontend/src/components/results/PlayerResultsItem.tsx

```diff
@@ -159,6 +161,16 @@ function PlayerResultsItem(props: PlayerResultsCardProps) {
         <Text>{getSubmissionCount()}</Text>
       </td>
       <CodeColumn>{getSubmissionLanguage()}</CodeColumn>
+      {onSpectateLive ? (
+        <CodeColumn>
+          <PreviewContainer>
+            <TextButton onClick={onSpectateLive}>
```

Comment:
I think that makes sense - I'll make that change such that if the `onSpectateLive` function is passed in, then it will only display the "Live Spectate" column and not the "Preview Code' column.

Reply:
Done! ✅